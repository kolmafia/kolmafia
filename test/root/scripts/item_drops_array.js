const { print, toJson, itemDropsArray } = require('kolmafia');

let monster = Monster.get("beanbat");
let drops = itemDropsArray(monster);

// Rhino doesn't support template literal strings. Alas.
print(monster + " has " + drops.length + " drops.");
print("The first drop is " + drops[0].drop + " with a rate of " + drops[0].rate + ".");
print("Its image is " + drops[0].drop.image + ".");
