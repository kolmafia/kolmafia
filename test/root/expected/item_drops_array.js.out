beanbat has 2 drops.
The first drop is enchanted bean with a rate of 50.
Its image is bean.gif.
